### ISSUE TITLE
Installation via brew?

### ISSUE THREAD

**Issue by ratkins** [MakeAWishFoundation/SwiftyMocky#276]
If `swiftymocky` were installable via `homebrew` we could remove our workstation/ci setup scripts' dependency on `mint` (which requires a non-interactive `sudo` command to create and set ownership on its path on M1 Macs.)

Is there any particular reason there's no `swiftymocky` formula in `brew`, aside from "nobody's built it yet"?

**Comment by wyszo** [comment]
There's a ticket open for that in the SwiftyMocky CLI repository: https://github.com/MakeAWishFoundation/SwiftyMockyCLI/issues/2

AFAIK 'nobody's built it yet' seems to be the main reason why it's not done. 

I had a quick look the other day and failed to build swiftymocky from a custom homebrew formula, so all I know is that it's not completely trivial (but it should be doable). 

It's possible that the formula will have to live in a tap (as opposed to the official brew repository), but that'd still be pretty great if we get to that stage.

**Comment by g-amichnia** [comment]
@wyszo Yes, it would be a tap probably. We have Makefile, I think I've seen using that for brew tap builds. And yes, "nobody built it yet" is actual reason :) 

@ratkins have you tried using make to drop using mint? (https://cdn.rawgit.com/MakeAWishFoundation/SwiftyMocky/4.0.3/docs/installation.html) Just in that case you would need to also manually install sourcery (not sure how up to date is brew version of it), and specify sourceryCommand in Mockfile (by default it uses mint, as it is for now recommend way of installing CLI).

**Comment by ratkins** [comment]
Getting rid of `mint` is the point for me, I’ll see if I have the chance to take a look at making a tap.

**Comment by isaac-weisberg** [comment]
Well damn

**Comment by spaluchiewicz** [comment]
@g-amichnia / @amichnia  I created **Homebrew Formula** for swiftymocky to install it with `brew` as described in this ticket.
Could you please review it: https://github.com/spaluchiewicz/homebrew-swiftymocky 
later we'll probably move it to `MakeAWishFoundation/SwiftyMocky`

Usage:
**NOTE:** Before you start installing swiftymocky with `brew` please remember to remove current version installed with `mint`
```
brew tap spaluchiewicz/homebrew-swiftymocky
brew install swiftymocky
```

**Comment by wyszo** [comment]
@spaluchiewicz In your tap I think you're missing `depends_on "sourcery"`. Without it I doubt it'll install and run properly on a clean system (other than that: nice work!)

**Comment by spaluchiewicz** [comment]
@wyszo I can add this but it looks like it's not necessary or it depends on your configuration:
- no `Sourcery` installed:
```bash
❯ brew list
==> Formulae
aom			freetype		htop			libde265		libpng			m4			openssl@1.1		swiftymocky
brotli			gdbm			icu4c			libffi			libtiff			mpdecimal		pcre			webp
c-ares			gettext			imagemagick		libheif			libtool			ncurses			python@3.9		wget
ca-certificates		gh			imath			libidn			libunistring		nmap			readline		x265
docbook			ghostscript		jbig2dec		libidn2			libuv			node			ruby			xmlto
docbook-xsl		giflib			jpeg			liblqr			libvmaf			openexr			shared-mime-info	xz
docker			glib			jpeg-xl			libnghttp2		libyaml			openjdk@11		speedtest		zsh-syntax-highlighting
fontconfig		gnu-getopt		lcov			libomp			little-cms2		openjpeg		sqlite

==> Casks
bitwarden	fork		iterm2		rectangle
```
but if you run `swiftymocky generate` in project where `SwiftyMocky` installed as dependency with `Cocoapods` (`Sourcery` installed as dependency for `SwiftyMocky` with Cocoapods) `SwiftyMockyCLI`will use `Sourcery` from `Pods`.

Will investigate this deeper.

**Comment by spaluchiewicz** [comment]
Done! I didn't find the need to install/set **Sourcery** as a dependency for CLI. 

@g-amichnia / @amichnia waiting for final check on your.
